contracts/Core/VoteManager.sol

```diff
@@ -33,6 +57,20 @@ contract VoteManager is Initializable, VoteStorage, StateManager, VoteManagerPar
         collectionManager = ICollectionManager(collectionManagerAddress);
     }
 
+    /**
+     * @notice stakers query the jobs in each collection, aggregate and instead of revealing them instantly,
```

Comment:
hmm this is not true
you don't have to fetch for each collection 
Only for assigned ones 

also next line tho its corrected line after
they don't submit hash of their results 
```
hash of their results which becomes their commitment 
```
The correct flow is you fetch numActiveCollections, find assignments, fetch those results, construct Merkle Pos aware tree, commit hash of root and seed

Reply:
Will remove the word `each` here but the thought process here was to write a generic statement of what happens in commit and in dev, mention the technical parts of the committing